src/memory_manager.rs

```diff
@@ -294,6 +291,8 @@ impl<M: Memory> Memory for VirtualMemory<M> {
 struct MemoryManagerInner<M: Memory> {
     memory: M,
 
+    version: u8,
```

Comment:
Do we need this?

Reply:
It is only used in the `upgrade_from_v1_to_v2` test to ensure the correct version of `grow` is called.
I'll wrap it in a `cfg(test)`.